app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class MessagesController < ApplicationController
+  def index
+    @message = Message.first
+    render json: @message, status: :ok
+  end
+end
```

Comment:
- [ ] You're actually required to create an action here that would generate random greetings, not the first one alone. Something like this. :point_down: 
```
     def index
         random_message = Message.all.sample
         @message = random_message
         render json: @message, status: :ok
     end

    OR

      def index
         @message = Message.all.sample
          render json: @message, status: :ok
       end
``` 

With your code, it would only generate the first greeting.

Reply:
Done :heavy_check_mark: